classes/05class/make/README.md

```diff
@@ -0,0 +1,185 @@
+# Make
+This class is a general overview of the Make utility and the Makefile.
+The main goal is to explain the purpose, the benefits and how to use it.
+
+***Contents***
+- [What is Make](#what-is-make)
+- [Why use Make?](#why-use-make)
+- [Makefile](#makefile)
+- [Running some code](#running-some-code)
+    - [Multiple Targets](#multiple-targets)
+    - [Pre-Requisites](#pre-requisites)
+    - [Variables](#variables)
+- [Mixing together](#mixing-together)
+
+## What is Make?
+Make is a tool to execute a sequence of commands (like a recipe) based on targets defined in a specific file.
+Commonly used to compile and build applications from its source code, the make utility has became very popular in the DevOps community, being used to create scripts for automation.
+
+## Why use Make?
+One of the biggest benefits of the Make utility is the ability to run in multiple platforms and shells. For example, if an engineer creates automations using bash scripts, another engineer using a different shell, like zsh, might have problems executing those scripts locally.
+
+By using make, the same script can be used in any Linux and MacOS operating system as long as the Make utility is installed. This approach improved the collaboration in DevOps/Cloud teams and made Make a very popular tool in the community.
+
+## Makefile
+The Makefile contains the targets and instructions that will be used when the Make utility is executed. It can contain variables, conditions (`if` and `else`) and loops (like `for`).
+
+As you can see the Makefile can be very complex and it can be used for multiple purposes. For this course and the pupose of what we'll be mostly using, we'll only cover the basics of it.
```

Comment:
```suggestion
As you can see the Makefile can be very complex and it can be used for multiple purposes. For this course and the purpose of what we'll be mostly using, we'll only cover the basics of it.
```